.github/workflows/release-artifacts.yml

```diff
@@ -65,9 +65,6 @@ jobs:
     - name: Build linux/amd64 Docker image
       uses: docker/build-push-action@v5
       with:
-        cache-from: type=gha
```

Comment:
While I understand the idea behind not caching release builds, I think we should leave the `cache-from` directive here. It can only speed up the process, and it does not have any drawbacks, afaik.

Reply:
The only drawback here, really, is that using `type=gha` for caching ends up filling the GitHub Actions cache with a bunch of artifacts that don't seem to be successfully used for caching.